benchmarks/Dapper.Tests.Performance/Dapper.Tests.Performance.csproj

```
@@ -47,5 +47,8 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' != 'net462'">
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.0" />
+    <PackageReference Include="Norm.net">
+      <Version>3.2.0</Version>
+    </PackageReference>
```

Tidying/normalizing here a bit!
```suggestion
    <PackageReference Include="Norm.net" Version="3.2.0" />
```